Added screenshots and Linux installation manual (#53)
eProsima/Fast-DDS-monitor: docs/rst/installation/linux.rst

#########################
Fast DDS Monitor on Linux
#########################

This section provides instructions on how to install the *Fast DDS Monitor* application.
This is available on the `eProsima <https://www.eprosima.com/>`_ website in the
`Downloads <https://www.eprosima.com/index.php/downloads-all>`_ section.

There are two mechanisms for running the monitor application.

> - Through the *Fast DDS Monitor* installer.

Reviewer: Add break line.
It is not well rendered.

- Using the *AppImage* format, which is a portable format of the application software.

*Fast DDS Monitor* installer
============================

The first option is to install the *Fast DDS monitor* application together with all its dependencies.

> To do this, first run the ``FastDDSMonitorInstaller.run`` executable and follow the instructions it ç

Reviewer: ```suggestion
To do so, first run the ``FastDDSMonitorInstaller.run`` executable and follow the instructions it
```

> provides to install the monitor in a directory on the system.

Reviewer: NIT

```suggestion
provides to install the program in a directory on the system.
```
